navigation/Navigation.tsx

```diff
@@ -119,7 +120,7 @@ export const Nav: React.FC<AppStackProps> = ({
           )}
         </Screen>
 
-        <Screen name="Maximize">
+        <Screen name="MaximizeScreen">
```

Comment:
Revert to Maximize

Reply:
Wdym ? It doesn't work if I put Maximise

Reply:
You need to add the necessary parameters. Look at how the navigation is done in the challenge component. There are a few this to change in the root StackParamList too. There must be an additional parameter for user added. This is why it doesn't work right now. If you pass the parameters like this the navigation with my type enforced, at the moment the navigation passes what ever it has but this could lead to some errors especially when testing. eg. End to end. If there is a ngl it will be passed down and will be harder to debug